lib/helpers.js

```diff
@@ -849,34 +849,26 @@ function toAvdLocaleArgs (language, country) {
  */
 async function getAndroidPrefsRoot () {
   // https://gerrit.pixelexperience.org/plugins/gitiles/development/+/5c11852110eeb03dc5a69111354b383f98d15336/tools/androidprefs/src/com/android/prefs/AndroidLocation.java
-  let location = null;
-  if (system.isWindows()) {
-    let localAppData = process.env.LOCALAPPDATA;
-    if (!localAppData) {
-      localAppData = process.env.USERPROFILE;
-      if (localAppData) {
-        localAppData = path.resolve(localAppData, 'Local Settings', 'Application Data');
-      }
-    }
-    if (localAppData) {
-      location = path.resolve(localAppData, 'Android');
-    }
-  } else {
-    const home = process.env.HOME;
+  let location = process.env.ANDROID_EMULATOR_HOME;
```

Comment:
we need to provide more detailed logs if ANDROID_EMULATOR_HOME and ANDROID_AVD_HOME are set to incorrect values, so a user could figure it out and fix their values if needed